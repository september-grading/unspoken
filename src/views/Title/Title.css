.title-container {
  display: flex;
  margin: auto;
  flex-direction: column;
  align-items: center;
}

.title-container h3 {
  color: white;
}
