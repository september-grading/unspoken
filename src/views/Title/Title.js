import React from 'react';
import Header from '../../components/Header/Header';
import Auth from '../../views/Auth/Auth';
import './Title.css';

export default function Title({ setCurrentUser }) {
  return (
    <div className="title-container">
      <img src="https://ipvfiyhmiwmzfnzlbhov.supabase.in/storage/v1/object/public/unspoken/unspoken_stories_header.png" />
      <Auth setCurrentUser={setCurrentUser} />
    </div>
  );
}
