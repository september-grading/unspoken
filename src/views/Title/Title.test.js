import { MemoryRouter, ProtectedRoute } from 'react-router-dom/cjs/react-router-dom.min';
import Title from '../Title/Title';
import { render, screen } from '@testing-library/react';

test.skip('should render title of post', async () => {
  const { container } = render(
    <MemoryRouter initialEntries={['/posts/37']}>
      <ProtectedRoute exact path="/posts/:id" component={Title} />
    </MemoryRouter>
  );
  await screen.findByText('Tokio and Nori');
  expect(container).toMatchSnapshot();
});
