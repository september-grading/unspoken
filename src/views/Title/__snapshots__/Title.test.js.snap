// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render title of post 1`] = `
<div>
  <div
    class="title-container"
  >
    <img
      src="https://ipvfiyhmiwmzfnzlbhov.supabase.in/storage/v1/object/public/unspoken/unspoken_stories_header.png"
    />
    <p>
      Write your stories, memories or unspoken words while listening to lofi beats. Sometimes all you need is to write it out.
    </p>
    <div>
      <span
        class="auth-tabs"
      >
        <h3
          class="active"
        >
          Sign In
        </h3>
        <h3
          class=""
        >
          Sign Up
        </h3>
      </span>
      <form
        class="auth-form"
      >
        
        <label>
          Email: 
        </label>
        <input
          type="email"
          value=""
        />
        <label>
          Password: 
        </label>
        <input
          type="password"
          value=""
        />
        <input
          class="submit-btn"
          type="submit"
        />
      </form>
    </div>
    <footer>
      <a
        class="about-btn"
        href="/about"
      >
        About Us
      </a>
    </footer>
  </div>
</div>
`;
